scalpel/flattening/flat_table.py

```diff
@@ -239,3 +240,52 @@ def _difference(a: FlatTable, b: FlatTable, join_keys: List[str]) -> FlatTable:
         [*a.join_keys],
         {**a.single_tables},
     )
+
+
+class HistoryTable(FlatTable):
```

Comment:
1. The History table represent flat table stat.
2. The flat table contains union method that will be used in building History table

So, I think it can inherit from flat table

Reply:
good idea